### PR TITLE
new githooks module

### PR DESCRIPTION
**Pull request by pcgentry**
This is a new module that lets you control service hooks in github.com via their api. 

Right now it just has two basic functions. It allows you to create a new service hook and it also has a function that removes hooks if the last status was something other than a 200 OK.

### CODE REVIEW

**Comment by jctanner** [comment]
@pcgentry thank you!

File: library/source_control/githooks [hunk 1]
```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
+import json
+import urllib2
+import base64
+
+"""
+Ansible module to add authorized_keys for ssh logins.
```
Comment (srgvg) [inline library/source_control/githooks:7]: sounds like copy pasted from another module?

File: library/source_control/githooks [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+DOCUMENTATION = '''
+---
+module: githooks
+short_description: Manages github service hooks.
+description:
+     - Adds service hooks and removes service hooks that have an error status.
+version_added: "1.3.4"
```
Comment (srgvg) [inline library/source_control/githooks:32]: shouldn't this be 1.4?

**Comment by jctanner** [comment]
@sergevanginderachter thanks! Fixed in dfe6c5d43472cbfb3288aba18808716204337ab2

**Pull request merged**